pkg/kubelet/kuberuntime/kuberuntime_image.go

```diff
@@ -32,11 +32,8 @@ import (
 // secrets if necessary.
 func (m *kubeGenericRuntimeManager) PullImage(ctx context.Context, image kubecontainer.ImageSpec, pullSecrets []v1.Secret, podSandboxConfig *runtimeapi.PodSandboxConfig) (string, error) {
 	img := image.Image
-	repoToPull, _, _, err := parsers.ParseImageName(img)
-	if err != nil {
-		return "", err
-	}
-
+	// Ignore error since we are validating the image names when creating these pods.
```

Comment:
Also, how would this work for static pods? When the validation be performed?

Reply:
I reverted these changes and left all validation in.